### PR TITLE
#4 Settings does not exist

### PR DESCRIPTION
**Pull request by AVATOR**

### CODE REVIEW

File: Service/Settings.php [hunk 1]
```diff
@@ -48,7 +52,12 @@ private function getCacheGroupKey($name)
 
     private function fetch($name)
     {
-        return $this->repository->findOneBy(array('name' => $name))->getValue();
+        $value = $this->repository->findOneBy(array('name' => $name));
+        if ($value) {
+            return $value->getValue();
+        }
+
+        return "";
```
Comment (lexxpavlov) [inline Service/Settings.php]: must be `return null;`

File: Service/Settings.php [hunk 2]
```diff
@@ -98,19 +107,24 @@ private function loadGroup($name)
      *
      * @param string $name Setting name or group name (if $subname is set)
      * @param string|null $subname Setting name (use with $name as group name)
+     * @param null $default The default value if the setting key does not exist
```
Comment (lexxpavlov) [inline Service/Settings.php]: Type of $default must be `mixed|null`

File: Service/Settings.php [hunk 3]
```diff
@@ -48,7 +52,12 @@ private function getCacheGroupKey($name)
 
     private function fetch($name)
     {
-        return $this->repository->findOneBy(array('name' => $name))->getValue();
+        $value = $this->repository->findOneBy(array('name' => $name));
```
Comment (lexxpavlov) [inline Service/Settings.php]: Variable name must be `$setting`

**Review by lexxpavlov — CHANGES_REQUESTED** [review]
It's a great job! But fix some minor changes, please.

**Comment by AVATOR** [comment]
I pushed changes.

**Review by lexxpavlov — APPROVED** [review]

**Pull request merged**